Python/ogusa/TPI.py

```diff
@@ -832,7 +834,10 @@ def TP_solutions(winit, rinit, T_H_init, BQinit2, Kss, Lss, Yss, BQss, theta, in
     eul_laborleisure = euler_errors[:, S:, :].max(1).max(1)
 
     print 'Max Euler error, savings: ', eul_savings
-    print 'Max Euler error labor supply: ', eul_laborleisure
+    print 'Max Euler error labor supply: ', eul_savings
```

Comment:
@jdebacker this is a typo, correct? I think the original line:

```
print 'Max Euler error labor supply: ', eul_laborleisure
```

should stay. Do you agree?